### PR TITLE
[feat] add flux pipeline to v1

### PR DESCRIPTION
**Pull request by Eigensystem**
- [x] add pipelines and modules
- [x] debug accuracy issues
- [x] refactor some hard code in stages
- [x] refactor custom stages & remove FluxDenoiseStage & OutputStage
- [x] add a `image_generate` func in `VideoGenerator`

### CODE REVIEW

File: fastvideo/v1/tests/transformers/test_flux.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        torch_dtype=precision).to(device, dtype=precision).requires_grad_(False)
+
+    total_params = sum(p.numel() for p in model1.parameters())
+    # Calculate weight sum for model1 (converting to float64 to avoid overflow)
+    weight_sum_model1 = sum(
+        p.to(torch.float64).sum().item() for p in model1.parameters())
+    # Also calculate mean for more stable comparison
+    weight_mean_model1 = weight_sum_model1 / total_params
+    logger.info("Model 1 weight sum: %s", weight_sum_model1)
+    logger.info("Model 1 weight mean: %s", weight_mean_model1)
```
Comment (Edenzzzz) [inline fastvideo/v1/tests/transformers/test_flux.py:56]: I feel there's no need to print the sums or means, only the diffs are useful

Comment (Eigensystem) [reply fastvideo/v1/tests/transformers/test_flux.py:56]: Okay. I wrote this test by referring to another file, fastvideo/v1/tests/transformers/test_wanvideo.py. Do I also need to remove the sum printing in this file as well?

Comment (Edenzzzz) [reply fastvideo/v1/tests/transformers/test_flux.py:56]: Possibly, it's more of a style thing, but most people just use `torch.testing.assert_close` to avoid printing too messily😂

**Comment by Eigensystem** [comment]
move to #419
